### PR TITLE
OSDOCS-3161 Add Compliance benchmark column to Supported profiles table

### PR DESCRIPTION
**Pull request by kmccarron-rh**
https://issues.redhat.com/browse/OSDOCS-3161
Applies to 4.6+

Direct link to the Supported profiles table [here](https://deploy-preview-40531--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-supported-profiles.html)

### CODE REVIEW

**Comment by kmccarron-rh** [comment]
Looking for lgtm so that we can move this PR forward. Please respond with comments or your ok. thank you!
@rhdcaspin @nkinder for dev
@pdhamdhe  for QE

**Comment by nkinder** [comment]
lgtm!

**Comment by kmccarron-rh** [comment]
@nermina-redhat - Will you also PTAL?

**Comment by rhmdnd** [comment]
Looks good to me

**Comment by nermina-redhat** [comment]
LGTM! Nice work, Karen!

File: modules/compliance-supported-profiles.adoc [hunk 1]
```diff
@@ -8,60 +8,72 @@
...
+|Profile
 |Profile title
 |Compliance Operator version 
+|Compliance benchmark
 
 |ocp4-cis
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Red Hat OpenShift Container Platform 4 Benchmark]
+|CIS Red Hat OpenShift Container Platform 4 Benchmark
 |0.1.39+
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] footnoteref:[fn1,To locate the CIS RedHat OpenShift Container Platform v4 Benchmark, go to  link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks] and type `Kubernetes` in the search box. Click on Kubernetes > Download Latest CIS Benchmark, where you can then register to download the benchmark.]
```
Comment (nalhadef) [inline modules/compliance-supported-profiles.adoc:22]: I think "Kubernetes > Download Latest CIS Benchmark" should be bold (https://www.ibm.com/docs/en/ibm-style?topic=elements-ui). If this doesn't fit the referenced example, I'd reword to "Click on **Kubernetes** and then **Download Latest CIS Benchmark**."

**Comment by nalhadef** [comment]
Karen, consider renaming the new column to "Industry compliance benchmark" to reinforce that these are external links.

File: modules/compliance-supported-profiles.adoc (hunk 1 above)
Comment (kmccarron-rh) [reply modules/compliance-supported-profiles.adoc]: Done
